centreon/src/Core/Dashboard/Application/UseCase/FindDashboard/FindDashboardResponse.php

```diff
@@ -32,7 +32,7 @@ final class FindDashboardResponse
 
     public string $name = '';
 
-    public string $description = '';
+    public ?string $description = null;
```

Comment:
Why this change ? Any particular reason coming from a frontend request ?

Reply:
There is basically no need to handle this as an empty string, the description is a nullable parameter so it should be nullabled, althought the empty string were converted in empty string into the repository